app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Forward.kt

```diff
@@ -5,44 +5,74 @@ import androidx.compose.foundation.layout.size
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.graphics.PathFillType
+import androidx.compose.ui.graphics.PathFillType.Companion.EvenOdd
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
 import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
 import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.group
 import androidx.compose.ui.graphics.vector.path
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import com.aptoide.android.aptoidegames.theme.Palette
 
 @Preview
 @Composable
 fun TestForward() {
   Image(
-    imageVector = getForward(Color.Black),
+    imageVector = getForward(Palette.Grey, Palette.AppCoinsPink),
     contentDescription = null,
     modifier = Modifier.size(240.dp)
   )
 }
 
-fun getForward(iconColor: Color): ImageVector = ImageVector.Builder(
-  name = "Forward",
-  defaultWidth = 24.dp,
-  defaultHeight = 24.dp,
-  viewportWidth = 24f,
-  viewportHeight = 24f,
+fun getForward(
+  backgroundColor: Color = Palette.Primary,
```

Comment:
Let's follow the convention we have about colors by naming them as color1, color2 etc.

Reply:
Fixed